packages/routing-utils/test/superstatic.spec.js

```diff
@@ -564,7 +584,7 @@ test('convertRewrites', () => {
     { src: '^\\/some\\/old\\/path$', dest: '/some/new/path', check: true },
     {
       src: '^\\/proxy(?:\\/(.*))$',
-      dest: 'https://www.firebase.com',
+      dest: 'https://www.firebase.com/',
```

Comment:
Can you add a test where the input has a trailing slash? Just want to make sure it doesn’t double up.

Reply:
Added